app/mailers/mailer.rb

```diff
@@ -17,4 +17,10 @@ def email_confirmation(user)
          subject: I18n.t('mailer.confirmation_subject',
            default: 'Please confirm your email address with RubyGems.org')
   end
+
+  def deletion_confirmation(email)
```

Comment:
`confirmation` sounds like you need to confirm something to me. If I understand this well, this is just info email.